### ISSUE TITLE
question-words.txt link broken for word2vec

### ISSUE THREAD

**Issue by andreiciceu** [tensorflow/tensorflow#2568]
The link for the word2vec analogy-question text file is borken:
[https://word2vec.googlecode.com/svn/trunk/questions-words.txt](https://word2vec.googlecode.com/svn/trunk/questions-words.txt)

**Comment by woidda** [comment]
You can download the archived source code of the original code and extract a single file by:

```terminal
wget https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/word2vec/source-archive.zip
unzip -p source-archive.zip  word2vec/trunk/questions-phrases.txt > questions-phrases.txt
rm source-archive.zip
```

**Comment by astromme** [comment]
This link is still present in master at https://github.com/tensorflow/tensorflow/blob/r0.10/tensorflow/models/embedding/word2vec_optimized.py#L54

**Comment by aselle** [comment]
Fixed internally, will be sync'd soon.

**Comment by astromme** [comment]
Thanks!

**Comment by aselle** [comment]
This is fixed as of 1c131f29d7877e3155f1e58ef3aa99b51faf399b
